sql/src/main/java/io/crate/operation/projectors/IndexWriterProjector.java

```diff
@@ -145,11 +138,15 @@ public Result setNextRow(Row row) {
 
     @Override
     public void finish(RepeatHandle repeatHandle) {
+        BulkProcessorFutureCallback bulkProcessorFutureCallback = new BulkProcessorFutureCallback(failed, downstream);
```

Comment:
also in this file, are they related to the commits? if not, make a separate commit pls.